library/network/src/modules/SuSEFirewall.rb

```diff
@@ -447,6 +447,30 @@ def GetModified
       @modified || SuSEFirewallServices.GetModified
     end
 
+    # By default Firewall packages are just checked whether they are installed.
+    # With this function, you can change the behavior to also offer installing
+    # the packages.
+    #
+    # @param [Boolean] new_status, 'true' if packages should be offered for installation
+    def SetInstallPackagesIfMissing(new_status)
+      if new_status.nil?
+        Builtins.y2error("Wrong value: %1", new_status)
+        return
+      end
+
+      @check_and_install_package = new_status
+
+      if @check_and_install_package
+        Builtins.y2milestone("Firewall packages will installed if missing")
+      else
+        Builtins.y2milestone(
+          "Firewall packages will not be installed even if missing"
+        )
+      end
+
+      nil
```

Comment:
The commit message explains that the method can be moved but in fact it was copied. Please remove the original.